OG_Format.adoc

```diff
@@ -219,37 +219,39 @@ Example 2 :
 
 Example 3 :
 
-* platform :
+* platform : SG669-20210617 
```

Comment:
In example 1 you use ` = ` 
i.e. `Programme = MOOSE glider program`

and in example 2, 3 ... you use ` : `
`Programme : NOAA Hurricane Glider program`

I suggest to harmonize these examples before merging.

Reply:
I will address this in a separate pull-request covering the whole document